### DISCUSSION TITLE
NAS Final Directory Option Gone

### DISCUSSION THREAD

**Discussion by hero887** [Chia-Network/chia-blockchain#6525] [Support]
Hi, it appears 1.1.7 no longer gives me the option to set my NAS as a final directory for plots. Is there a setting I am missing or do NAS not work with Chia as a final directory now?

**Comment by loppefaaret** [comment]
It should show up - try and remount the share, and restart Chia

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
